### PR TITLE
Feat: LLM playground improvements

### PR DESCRIPTION
**Pull request by synapse**
- [x] Added compiled `dist` folder
- [x] Exposed `getDocuments` method which lists all the files in the database given a filter
- [x] Exposed `getDocument` method which returns 1 file
- [x] Various prompt fixes and query improvements

### CODE REVIEW

**Review by simoneb — COMMENTED** [review]
please check linting errors

**Comment by marceloFerreira90** [comment]
exclude dist from git we can build it on ci  is better

File: src/db/documents.ts [hunk 1]
```diff
@@ -60,6 +60,37 @@ export async function getDocument(
...
 
+export async function getDocuments(
+  connPool: pg.Pool,
+  filters?: Record<string, string | string[]>
+): Promise<Document[]> {
+  const client = await connPool.connect()
+
+  const conditionArray: SqlStatement[] = []
+
+  if (filters != null) {
```
Comment (marceloFerreira90) [inline src/db/documents.ts:71]: if(!filters) is better to use here

**Comment by simoneb** [comment]
if we exclude this from git, how can a consumer use the transpiled output? if we were releasing an npm package, yes we could do the build in CI and publish that package with the output included, but since we're referencing this as a git repo, we can't do that

**Review by marceloFerreira90 — COMMENTED** [review]
1 small change

 one bigger one
 
 if you need help on the ci reach out.

File: src/db/documents.ts (hunk 1 above)
Comment (simoneb) [reply src/db/documents.ts:71]: I agree, it's not idiomatic js to compare against explicit falsy values unless you really need to check against a specific value. here we're only interested in whether filters are there or not. the code, written in this way, is unnecessary coupled and less robust

Comment (synapse) [reply src/db/documents.ts:71]: `if (filters)` ?

**Pull request merged**